### DISCUSSION TITLE
SSH default credentials

### DISCUSSION THREAD

**Discussion by akerblom** [bareboat-necessities/lysmarine_gen#394] [Q&A] [answered]
Hi, 
I'm a newbie at this and I just installed BBN-OS on my RPI 5.  No I want to access it all from my pc via PuTTY. But what is the default credentials? The manual say "You can log in using ssh user: 'user'"... This is probably obvious but what is the username and password?

**Comment by mgrouch** [comment, marked as answer]
User is ‘user’. Password is ‘changeme’. It’s in the documentation

**Reply by akerblom** [reply]
Thank you! Works perfectly :-)
